<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Potvrď</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>188</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>9</number>
    </property>
    <property name="verticalSpacing">
     <number>16</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Kód studenta</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Jméno studenta</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_2</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Body historie</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_3</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Body zeměpis</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_4</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Celkem</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Úspěšnost</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_total">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_perc">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>disp_message()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>disp_message()</slot>
 </slots>
</ui>
